packages/core/content-type-builder/admin/src/components/FormModal/attributes/types.js

```diff
@@ -156,10 +158,23 @@ const types = {
           },
         })
         .test({
-          name: 'valuesMatchesRegex',
-          message: errorsTrads.regex,
-          test: values => {
-            return values.every(val => val === '' || ENUM_REGEX.test(val));
+          name: 'valuesCollide',
+          message: 'Some values collide when normalized',
+          test(values) {
+            const normalizedEnum = values.map(toGraphQLName);
```

Comment:
This will change what the user has entered in the enum textarea (on save), not sure it's what we want 🤔 ?

Reply:
I'm not sure how it would change it? It's not the purpose indeed ahah